src/Obfuscatories/SymmetricObfuscatorInterface.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace RobThree\HumanoID\Obfuscatories;
+
+interface SymmetricObfuscatorInterface
```

Comment:
> SymmetricObfuscatorInterface

I'm not totally in love with the naming yet - I just picked what felt closest from our ideas.

In this context I used "Symmetric" to mean that w/e technique is used, must be reversible; rather then meaning that a single method must obfuscate and unobfuscate as we once discussed.

Reply:
I understood what you meant with symmetric immediately. And it makes it clear that this is not hashing but rather a form of (non secure obviously) cryptography that is reversible. I think it's a great name to use.